antlr-kotlin-runtime/src/commonMain/kotlin/com/strumenta/kotlinmultiplatform/Collections.kt

```diff
@@ -1,9 +1,9 @@
 package com.strumenta.kotlinmultiplatform
 
 public object Collections {
-  public fun <T : Comparable<T>> min(precedencePredicates: List<T>): T =
-    precedencePredicates.minOrNull() ?: throw NoSuchElementException()
+  public fun <T : Comparable<T>> min(collection: Collection<T>): T =
```

Comment:
collection is a bit of vague name, something like `precedencePredicates` was more clear, imho

Reply:
@ftomassetti but this can be used for whatever collection, not only for `precedencePredicates`.

Reply:
Ok, I see